Spigot-Server-Patches/0319-Add-more-Zombie-API.patch

```diff
@@ -29,24 +29,24 @@ index 12f954723ac2f0058081d810ce83aed99532caad..02a845f99bb445fb4aa66a90273469a8
      public void startDrownedConversion(int i) {
          this.lastTick = MinecraftServer.currentTick; // CraftBukkit
          this.drownedConversionTime = i;
-@@ -226,10 +233,17 @@ public class EntityZombie extends EntityMonster {
+@@ -226,9 +233,16 @@ public class EntityZombie extends EntityMonster {
  
      }
  
 +    public boolean shouldBurnInDay() { return T_(); } // Paper - OBFHELPER
      protected boolean T_() {
-         return true;
-     }
- 
+-        return true;
++        return this.shouldBurnInDay; // Paper - use api value instead
++    }
++
 +    // Paper start
 +    public void setShouldBurnInDay(boolean shouldBurnInDay) {
 +        this.shouldBurnInDay = shouldBurnInDay;
-+    }
+     }
 +    // Paper end
-+
+ 
```

Comment:
I don't really understand why is there so many changes here - it should be just:
```diff
  protected boolean T_() {
-   return true;
+   return this.shouldBurnInDay; // Paper - use api value instead
  }
```

Reply:
I only modified the one line. I guess that the other "changed" lines are caused by the usual heuristics that git performs in order to calculate diffs. It changed which brace it used/where it injects the patch content into the file. But I did not change any line except for the return.